src/scss/components/events/_event-node.scss

```diff
@@ -136,14 +133,15 @@
     }
 
     .su-event-sponsor {
-      @include responsive-spacing('padding-top', 0);
-
+      
       max-width: 100%;
       text-align: center;
       background: white;
       margin: 0 auto;
-
+      
```

Comment:
Is this an extra tab space? It is good to remove those.

Reply:
Thanks @jenbreese! Updated.